src/core/toga/app.py

```diff
@@ -415,6 +416,33 @@ def startup(self):
 
         self.main_window.show()
 
+    def about_command(self, widget):
```

Comment:
Nitpicking, but as this is going to be a public API, we need to be careful about the name. The `_command` suffix implies this will be (or return) a command, but it's actually the *action* that will be performed. `about` or `on_about` (and similar for homepage and quit) would make sense to me, but I'm open to other suggestions.